README.md

```diff
@@ -3,91 +3,83 @@
+- `ppafm-generate-elff-point-charges` - command-line interface to gelerate electrostatic force field using point charges.
+- `ppafm-generate-ljff` - command-line interface to gelerate Lennard-Jones force field.
+- `ppafm-relaxed-scan` - command-line interface to run scan the sample with the probe particle.
+- `ppafm-plot-results` - command-line interface to plot the results of the simulation.
+- `ppafm-gui` - GUI application for interactive simulation of AFM images.
 
-All development and testing were done on **linux** OS (mostly ubuntu). However, there are also pip wheels for Windows available. See also the docker image below for a platform-independent way of running the code.
 
-#### Install & run CPU version
+### Install GPU GUI
 
-**Requirements:** Python3 (numpy,matplotlib) & C/C++ compiler (g++,make)
+To make sure the `ppafm-gui` command works without problems, you need to install QT5 library on your system.
+On Ubuntu, you can do this by running:
 
-##### First run: Graphene with point-charges
- 1. clone the repository: `clone https://github.com/Probe-Particle/ProbeParticleModel.git`
- 2. compile the C/C++ modules
-    * `cd ProbeParticleModel/Graphene`
-    * `make`
- 3. Navigate to examples directory `cd ProbeParticleModel/examples/Graphene`this example uses simple (Point-charges + Lennard-Jones)
- 4. run the example `./run.sh`
- 5. output directory `/examples/Graphene/Q-0.05K0.50/Amp2.0` should contain simulated images with tip charge -0.05e, stiffness 0.5N/m and ossicaltion amplitude 2.0A.
+```bash
+$ sudo apt install git python3-pip python3-pyqt5
+```
 
-*NOTE:* Python package is designed to automatically recompile the C/C++ automatically, which is convenient for development, so explicit compilation in step #2 maybe not be necessary. see e.g. `cpp_utils.make("PP")` in `ppafm/core.py`
+The other dependencies should be installed automatically when you install the `ppafm` package with `opencl` option:
 
-##### Example 2: PTCDA with Hartree potential
+```bash
+$ pip install ppafm[opencl]
+```
 
-1. navigate to `ProbeParticleModel/examples/PTCDA_Hartree`
-2. run the example `./run.sh`
+Additionally an OpenCL Installable Client Driver (ICD) for your compute device is required:
+* Nvidia GPU: comes with the standard Nvidia driver (nvidia-driver-xxx)
+* AMD GPU: `sudo apt install mesa-opencl-icd`
+* Intel HD Graphics: `sudo apt install intel-opencl-icd`
+* CPU: `sudo apt install pocl-opencl-icd`
 
-*NOTE:* Notice that the script `run.sh` downloads and unpack LOCPOT file:
-`wget --no-check-certificate "https://www.dropbox.com/s/18eg89l89npll8x/LOCPOT.xsf.zip"`
-`unzip LOCPOT.xsf.zip`
-this is a large 3D volumetric file which contains Hartree electrostatic potential (in this example computed by VASP) which have to be provided.
+### Use ppafm Docker container
 
-##### Example 3: Pyridine with Density-overlap
+We propose to use [Docker](https://docs.docker.com/get-docker/) to make the code platform-independent.
```

Comment:
But we also plan to have wheels for all platforms, which should become the more straightforward installation option for every platform. Docker is useful to have as an option, but in order to keep the README minimal, maybe this should also be moved to the wiki page with additional installation options.

Reply:
Since you proposed to make follow-up PRs, will you take care of that too?

Reply:
Sure, I'll do it.